tests/scripts/auto-grow-storage.sh

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env bash
```

Comment:
This script needs to be linted I see a few white spaces, also please rebase so that the new shellcheck action can inspect it.

Reply:
Shellcheck action runs fine now:)